pkg/report/report.go

```diff
@@ -69,12 +69,19 @@ type Report struct {
 	GuiltyFile string
 	// Arbitrary information about the test VM, may be attached to the report by users of the package.
 	MachineInfo []byte
+	// If the crash happened in the context of the syz-executor process, SyzExecutor will hold more info.
+	SyzExecutor *ExecutorInfo
```

Comment:
Let's avoid prefixing things with "syz" unless absolutely necessary. Let's call it just Executor.
And similarly for syzLinuxCommRe and setSyzExecutor.
The binaries are prefixed with "syz-" just because manager/executor/fuzzer are too common names to conflict with, but we don't have this problem in the source code.